07 Data Library/04 Options/03 Requesting Options Data - OptionChainProvider.html

```diff
@@ -0,0 +1,49 @@
+<p>
+Another way to subscribe to options data is by using OptionChainProvider. In backtesting, OptionChainProvider allows you to request a list of options contracts for a specific symbol and a specific date. In Live Mode, the date argument is fixed to the current algorithm time.
```

Comment:
Shuffle to explain the basic concept first then in second paragraph for any differences in live and backtesting.